### PR TITLE
Feature/md 2136

### PR DESCRIPTION
**Pull request by stuisme**
Sticking with this markup for the export endpoint, apimatic does not currently support this endpoint.

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -70,10 +73,58 @@ Any response that does not provide a ```200 OK``` response code should be treate
...
+
 # Data Structures
 
 ## Heartbeat (object)
 Represents a heartbeat response.
 
 ### Properties
-+ `vitalCheck`: true (boolean, required) - Vital check to ensure something is online.
\ No newline at end of file
++ `vitalCheck`: true (boolean, required) - Vital check to ensure something is online.
```
Comment (brianhyder) [inline apiary.apib]: i would add the respective properties for the heartbeat.

**Comment by brianhyder** [comment]
```
╱╱┏╮
╱╱┃┃
▉━╯┗━╮
▉┈┈┈┈┃
▉╮┈┈┈┃
╱╰━━━╯
```

**Comment by stuisme** [comment]
Updated heartbeat docs from template.

File: apiary.apib [hunk 2]
```diff
@@ -54,6 +58,7 @@ None at this time
 In Progress
 
 # Group Heartbeat
+Check health of dependant services
```
Comment (elizabethmidgett) [inline apiary.apib:61]: dependent*

**Comment by brianhyder** [comment]
@stuisme Did you mean to close the issue or merge the issue?

**Comment by elizabethmidgett** [comment]
@brianhyder this was going to master. He opened a new one to orange that I merged.

**Pull request closed without merging**